tasks/src/test/java/taf/service/DocumentCreator.java

```diff
@@ -6,15 +6,10 @@
   public static final String DOCUMENT_BODY_TEXT = "testdata.document.text";
   public static final String DOCUMENT_NAME = "testdata.document.name";
 
-  public static Document getDocumentName() {
+  public static Document getDocumentInfo() {
     Document document = new Document();
     document.setName(TestDataReader.getTestData(DOCUMENT_NAME));
-    return new Document(document.getName());
-  }
-
-  public static Document getBodyText() {
-    Document text = new Document();
-    text.setText(TestDataReader.getTestData(DOCUMENT_BODY_TEXT));
-    return new Document(text.getText());
+    document.setText(TestDataReader.getTestData(DOCUMENT_BODY_TEXT));
+    return new Document(document.getName(), document.getText());
```

Comment:
Why don't you just return an existing document object initialized in the 10th line?